### ISSUE TITLE
Off-boarding Sheet

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-data-register#514]
#### Description:
This ticket is to have the essential knowledge transfer in place before the end of the internship.

#### Acceptance Criteria:
If Kavi were to leave tomorrow, what would be essential (according to her and the team members) to have a record of in the team's shared folder?

#### Development Checklist:
- [ ] Transfer a copy of my [Figjam board](https://www.figma.com/board/9oqz4wAbXdMHHpbuZV24Ii/Work-Notes?node-id=0-1&t=LWoQ0SsKLrQwKdol-1) in a shared place
- [ ] ...
- [ ] ...

#### Dependencies
Blocked by

Blocking
- [ ] Jess's task of deactivating my IDIR accounts

**Relevant documentation as reference**

**Definition of Ready**
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Work is traceable in GitHub

PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
@Dianadec - this ticket appears to be Kavi specific - assuming we can delete or close. Thoughts?

**Comment by LindsayMacfarlane** [comment]
Confirmed with Diana and Jess that this ticket is no longer needed. Closing.
